source/server/server.cc

```diff
@@ -139,6 +145,7 @@ void InstanceImpl::flushStats() {
   server_stats_->days_until_first_cert_expiring_.set(
       sslContextManager().daysUntilFirstCertExpires());
 
+  stats_store_.mergeHistograms();
   InstanceUtil::flushCountersAndGaugesToSinks(config_->statsSinks(), stats_store_);
```

Comment:
Can you rename this function to something more appropriate? `flushMetricsToSinks()` ?